### PR TITLE
FY-89/feat(core): add createElement and Fragment functions

### PR DESCRIPTION
**Pull request by daekuelee**
- Implemented createElement function in core/createElement.js
- Added Fragment function to support JSX fragments
- Created test cases for createElement and Fragment in tests/core/createElement.test.js

close #FY-89

## Overview

[FY-](https://idealflower.atlassian.net/ blahblah)

## PR Checklist

-   [ ] I read and included theses actions below

1. I have written documents and tests, if needed.

### CODE REVIEW

**Comment by daekuelee** [comment]
test error

File: srcs/core/createElement.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        },
+    };
+};
+
+const flatten = (arr) => {
+    return arr.reduce((acc, val) => {
+        if (val === null || val === undefined) {
+            return acc;
+        }
+        return isArray(val) ? acc.concat(flatten(val)) : acc.concat(val);
```
Comment (JinmuGo) [inline srcs/core/createElement.js]: isArray를 isArr로 변경해서 import하고 있는데, 이부분만 타이핑 수정해주시면 될 것 같습니다.

**Review by JinmuGo — CHANGES_REQUESTED** [review]
아래 내용을 확인하셔서 수정 바랍니다!

**Review by JinmuGo — APPROVED** [review]
수정 사항 확인하였습니다!

**Pull request merged**